packages/qdrant-loader-core/src/qdrant_loader_core/llm/providers/bedrock_utils.py

```
@@ -0,0 +1,169 @@
+    """Map a botocore/boto3 exception into a qdrant_loader_core LLMError."""
+    if isinstance(exc, NoCredentialsError):
+        return AuthError(str(exc))
+
+    error_code = ""
+    status_code = None
+    if hasattr(exc, "response"):
+        try:
+            error_code = exc.response.get("Error", {}).get("Code", "")
+            status_code = exc.response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)